jdaviz/configs/default/plugins/data_quality/data_quality.py

```diff
@@ -71,6 +72,41 @@ def __init__(self, *args, **kwargs):
 
         self.load_default_flag_maps()
         self.init_decoding()
+        self._set_irrelevant()
+        self._update_available_viewers()
+
+    def _update_available_viewers(self):
+        if not hasattr(self, 'viewer'):
+            return
+
+        def is_image_viewer(viewer):
+            from jdaviz.configs.imviz.plugins.viewers import ImvizImageView
+            from jdaviz.configs.cubeviz.plugins.viewers import CubevizImageView
+
+            return isinstance(viewer, (ImvizImageView, CubevizImageView))
```

Comment:
might be nice to move this into the `ViewerSelect` directly.  There already is an `is_image_viewer` defined, although it doesn't currently allow for cube viewers.  So maybe we can check the current uses for that to see if that can be extended to include either, or if we want a separate `is_image_or_cube_viewer`?